### ISSUE TITLE
Default simplify to use 'keep unary' option in tskit?

### ISSUE THREAD

**Issue by jeromekelleher** [tskit-dev/tsinfer#146]
Tskit now supports the ``keep_unary`` simplify option (https://github.com/tskit-dev/tskit/pull/143) which will probably be quite useful for tskit. This will hopefully give us a reduction in ts file size, and should probably be the default format to output.

Any thoughts @hyanwong?

**Comment by jeromekelleher** [comment]
If so, will depend on version 0.1.6 of tskit.

**Comment by jeromekelleher** [comment]
For current 1kg trees we get:
```
-rw-r--r--. 1 jk bioinf 997M Apr 12 13:47 1kg_chr1.full-simplify.trees
-rw-r--r--. 1 jk bioinf 859M Apr 12 13:42 1kg_chr1.keep-unary.trees                                                                             
-rw-r--r--. 1 jk bioinf 1.1G Apr 12 13:41 1kg_chr1.nosimplify.trees     
```
So we do get a decent reduction in file size from keep_unary.

**Comment by hyanwong** [comment]
Wow. I'm not understanding something here - I can't see how keeping extra nodes would *reduce* the file size, unless the extra nodes allow edges to be merged in some way. For example, in a slim simulation won't all unary nodes include a huge number of intermediate individuals (all the relevant ancestral individuals in every generation since the MRCA)?

**Comment by jeromekelleher** [comment]
Fully simplifying does add extra edges for tsinfer generated tree sequences because we're splitting up potentially long edges to where they happen to participate in coalescences. In some cases the unsimplified tree (containing everything) can be smaller than the simplified trees, but usually only when we're doing very badly anyway.

**Comment by jeromekelleher** [comment]
Closed in #176.
